src/bundle/Controller/Content/ContentTreeController.php

```diff
@@ -110,6 +122,82 @@ public function loadSubtreeAction(Request $request): Root
 
         return new Root($elements);
     }
+
+    public function loadNodeExtendedInfoAction(Request $request): NodeExtendedInfo
+    {
+        /** @var \Ibexa\AdminUi\REST\Value\ContentTree\LoadNodeExtendedInfoRequest $loadSubtreeRequest */
+        $loadSubtreeRequest = $this->inputDispatcher->parse(
+            new Message(
+                ['Content-Type' => $request->headers->get('Content-Type')],
+                $request->getContent()
+            )
+        );
+
+        $locationId = $loadSubtreeRequest->locationId;
+        $location = $this->locationService->loadLocation($locationId);
+        $locationPermissionRestrictions = $this->getLocationPermissionRestrictions($location);
+
+        return new NodeExtendedInfo($locationPermissionRestrictions);
+    }
+
+    /**
+     * @return PermissionRestrictions
+     *
+     * @throws \Ibexa\AdminUi\Exception\InvalidArgumentException
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\BadStateException
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\InvalidArgumentException
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException
+     */
+    public function getLocationPermissionRestrictions(Location $location): array
+    {
+        $lookupCreateLimitationsResult = $this->permissionChecker->getContentCreateLimitations($location);
+        $lookupUpdateLimitationsResult = $this->permissionChecker->getContentUpdateLimitations($location);
+        $lookupDeleteLimitationsResult = $this->permissionChecker->getContentDeleteLimitations($location);
+        $lookupHideLimitationsResult = $this->permissionChecker->getContentHideLimitations($location);
+
+        $createLimitationsValues = $this->lookupLimitationsTransformer->getGroupedLimitationValues(
+            $lookupCreateLimitationsResult,
+            [Limitation::CONTENTTYPE, Limitation::LANGUAGE]
+        );
+
+        $updateLimitationsValues = $this->lookupLimitationsTransformer->getGroupedLimitationValues(
+            $lookupUpdateLimitationsResult,
+            [Limitation::CONTENTTYPE, Limitation::LANGUAGE]
+        );
+
+        $deleteLimitationsValues = $this->lookupLimitationsTransformer->getGroupedLimitationValues(
+            $lookupDeleteLimitationsResult,
+            [Limitation::CONTENTTYPE, Limitation::LANGUAGE]
+        );
+
+        $hideLimitationsValues = $this->lookupLimitationsTransformer->getGroupedLimitationValues(
+            $lookupHideLimitationsResult,
+            [Limitation::CONTENTTYPE, Limitation::LANGUAGE]
+        );
+
+        return [
+            'create' => [
+                'hasAccess' => $lookupCreateLimitationsResult->hasAccess,
```

Comment:
With https://github.com/ibexa/core/pull/349:
```suggestion
                'hasAccess' => $lookupCreateLimitationsResult->hasAccess(),
```